config/limitador/redis-server.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: redisconfig
+  namespace: kuadrant-system
+stringData:
+  URL: redis://172.32.0.2:30611
+type: Opaque
```

Comment:
Consider using the kustomize secret generator (look for `secretGenerator`) for secrets and try and avoid using hard coded values specific to local setup in the base kustomizations.

Reply:
ok yes your right